src/components/products/product-apis/ko/runtime/product-apis.html

```diff
@@ -1,24 +1,70 @@
-<h3>APIs in the product</h3>
+<div class="row">
+    <div class="col-md-9">
+        <div class="search-input">
+            <input type="search" role="searchbox" aria-label="Search" placeholder="Search APIs"
```

Comment:
Should be "Search products" instead of "Search APIs".

Reply:
this is product's APIs